cmd/xeniad/midware/cayley.go

```diff
@@ -0,0 +1,53 @@
+package midware
+
+import (
+	"github.com/ardanlabs/kit/cfg"
+	"github.com/ardanlabs/kit/db"
+	kitcayley "github.com/ardanlabs/kit/db/cayley"
+	"github.com/ardanlabs/kit/log"
+	"github.com/ardanlabs/kit/web/app"
+
+	// mongo is needed to utilize mongoDB as the backend store for cayley.
+	_ "github.com/cayleygraph/cayley/graph/mongo"
+)
+
+const (
+	cfgMongoHost     = "MONGO_HOST"
+	cfgMongoUser     = "MONGO_USER"
+	cfgMongoPassword = "MONGO_PASS"
+)
+
+// Cayley handles session management.
+func Cayley(h app.Handler) app.Handler {
+
+	// Check if mongodb is configured.
+	mongoHost, err := cfg.String(cfgMongoHost)
+	if err != nil {
+		return func(c *app.Context) error {
+			log.Dev(c.SessionID, "Cayley", "******> Cayley Not Configured")
```

Comment:
I'd like to deprecate this style for logging personally. the `***` doesn't really add anything.